lib/reporting/protocols_report.rb

```diff
@@ -288,25 +274,30 @@ def loa_issuers_query
         | parse @message '"authn_context":[*]' as authn
         | filter
           name IN %{event}
-          AND (authn like /ns\\/assurance\\/loa/ OR acr like /ns\\/assurance\\/loa/)
+          AND (authn like /aal\\/3/ or acr like /aal\\/3/)
           AND properties.event_properties.success= 1
         | display issuer
         | sort issuer
         | dedup issuer
       QUERY
     end
 
-    def aal3_issuers_data
-      @aal3_issuers_data ||= cloudwatch_client.fetch(
-        query: aal3_issuers_query,
-        from: time_range.begin,
-        to: time_range.end,
-      ).
-        map { |slice| slice['issuer'] }.
-        uniq
+    def facial_match_issuers_query
+      params = {
+        event: quote([SAML_AUTH_EVENT, OIDC_AUTH_EVENT]),
+      }
+
+      format(<<~QUERY)
+        fields
+          coalesce(properties.event_properties.service_provider, properties.event_properties.client_id, properties.service_provider) as issuer
+        | filter properties.sp_request.facial_match
```

Comment:
Did you need to filter by event or does `properties.sp_request.facial_match` cover it? If the latter, you probably don't need `params`?

Reply:
don't need it! i just pushed a commit that removed it, thanks

Reply:
Never mind 😄